src/librustc_resolve/lib.rs

```diff
@@ -1271,15 +1273,62 @@ impl<'a> Resolver<'a> {
     }
 
     pub fn into_outputs(self) -> ResolverOutputs {
+        let definitions = self.definitions;
+        let extern_crate_map = {
```

Comment:
`HashMap` implements `FromIterator`. Why not use `collect` for all these like the old code does?

Reply:
Fixed!